rs/nns/governance/canister/canister.rs

```diff
@@ -161,15 +164,22 @@ fn set_governance(gov: Governance) {
 fn schedule_timers() {
     schedule_seeding(Duration::from_nanos(0));
     schedule_adjust_neurons_storage(Duration::from_nanos(0), NeuronIdProto { id: 0 });
+    schedule_prune_following(Duration::from_secs(0));
 }
 
 // Seeding interval seeks to find a balance between the need for rng secrecy, and
 // avoiding the overhead of frequent reseeding.
 const SEEDING_INTERVAL: Duration = Duration::from_secs(3600);
 const RETRY_SEEDING_INTERVAL: Duration = Duration::from_secs(30);
+const PRUNE_FOLLOWING_INTERVAL: Duration = Duration::from_secs(1);
```

Comment:
The base message instruction costs went up a lot, which means it might be better to have fewer of these, and then do more work in each one.

Reply:
IIRC from last GRnD, it went up 2x. I don't think that significantly throws off my back of the envelope calculations.

As long as these values are not harmful, I think we can just go with this for now, and adjust later after we get some real world experience with this.

If I understand [this graph][graph] correctly, 25 megainstructions per message is really on the high side.

[graph]: https://grafana.ch1-obs1.dfinity.network/d/execution-metrics/execution-metrics?var-heatmap_period=$__auto&from=now-30m&to=now&timezone=utc&var-datasource=PE62C54679EC3C073&var-ic=mercury&var-ic_subnet=$__all&var-instance=$__all&var-node_instance=$__all&viewPanel=panel-117

Reply:
If you think this doesn't work, I very very very welcome suggestions. Actionable good.

Reply:
https://forum.dfinity.org/t/evaluating-compute-pricing-in-response-to-increased-demand-on-the-internet-computer-protocol/36565

It's 9-10x message base fee (590k -> 5 million).

I'd suggest 50 million instructions per round, spaced every 10 seconds, which would only give you 1 pass per day (is my math right?)

Reply:
Done.